master/internal/rm/agentrm/provisioner/aws.go

```diff
@@ -31,6 +31,9 @@ type awsCluster struct {
 	spot *spotState
 }
 
+//nolint:lll
```

Comment:
Link to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html?